.github/workflows/release_docker.yml

```diff
@@ -11,6 +11,17 @@ jobs:
   docker:
     name: Push Docker image
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - image: ghcr.io/Lilypad-Tech/bacalhau
```

Comment:
The intention here is to make sure RPs have access to the specific version we are pinned to correct?

Reply:
> The intention here is to make sure RPs have access to the specific version we are pinned to correct?

Yes *and* so we can have an image for bacalhau compute nodes in more complex (k8s) setups for testing

Reply:
💪

Reply:
It seems `bacalhau` also publishes to the github registry and keeps specific version around ([reference](https://docs.bacalhau.org/getting-started/installation#step-1.1-install-the-bacalhau-cli))

Reply:
They *do* but it seems inconsistent? There's nothing tagged for `1.3.2` and when I first did this they hadn't done `1.4.0`... we can deprecate it in time?

Reply:
Yeah, I actually pinged them because their latest release actually return the `1.3.0` version 🤦